src/System.Memory/tests/Span/Sort.cs

```diff
@@ -0,0 +1,1606 @@
+        }
+        [OuterLoop]
+        [Theory]
+        [Trait(SortTrait, SortTraitValue)]
+        [MemberData(nameof(s_slowSortTests))]
+        public static void Sort_KeysValues_Single_Int32_OuterLoop(ISortCases sortCases)
+        {
+            Test_KeysValues_Single_Int32(sortCases);
+        }
+        [OuterLoop]
+        [Theory]
+        [Trait(SortTrait, SortTraitValue)]
+        [MemberData(nameof(s_slowSortTests))]
+        public static void Sort_KeysValues_Double_Int32_OuterLoop(ISortCases sortCases)
+        {
+            Test_KeysValues_Double_Int32(sortCases);
+        }
+        [OuterLoop]
+        [Theory]
+        [Trait(SortTrait, SortTraitValue)]
+        [MemberData(nameof(s_slowSortTests))]
+        public static void Sort_KeysValues_Boolean_Int32_OuterLoop(ISortCases sortCases)
+        {
+            Test_KeysValues_Boolean_Int32(sortCases);
+        }
+        [OuterLoop]
+        [Theory]
+        [Trait(SortTrait, SortTraitValue)]
+        [MemberData(nameof(s_slowSortTests))]
+        public static void Sort_KeysValues_Char_Int32_OuterLoop(ISortCases sortCases)
+        {
+            Test_KeysValues_Char_Int32(sortCases);
+        }
+        [OuterLoop]
+        [Theory]
+        [Trait(SortTrait, SortTraitValue)]
+        [MemberData(nameof(s_slowSortTests))]
+        public static void Sort_KeysValues_String_Int32_OuterLoop(ISortCases sortCases)
+        {
+            Test_KeysValues_String_Int32(sortCases);
+        }
+        [OuterLoop]
+        [Theory]
+        [Trait(SortTrait, SortTraitValue)]
+        [MemberData(nameof(s_slowSortTests))]
+        public static void Sort_KeysValues_ComparableStructInt32_Int32_OuterLoop(ISortCases sortCases)
+        {
+            Test_KeysValues_ComparableStructInt32_Int32(sortCases);
+        }
+        [OuterLoop]
+        [Theory]
+        [Trait(SortTrait, SortTraitValue)]
+        [MemberData(nameof(s_slowSortTests))]
+        public static void Sort_KeysValues_ComparableClassInt32_Int32_OuterLoop(ISortCases sortCases)
+        {
+            Test_KeysValues_ComparableClassInt32_Int32(sortCases);
+        }
+        [OuterLoop]
+        [Theory]
+        [Trait(SortTrait, SortTraitValue)]
+        [MemberData(nameof(s_slowSortTests))]
+        public static void Sort_KeysValues_BogusComparable_Int32_OuterLoop(ISortCases sortCases)
+        {
+            Test_KeysValues_BogusComparable_Int32(sortCases);
+        }
+#endif
+        #endregion
+
+        // NOTE: Sort_MaxLength_NoOverflow test is constrained to run on Windows and MacOSX because it causes
+        //       problems on Linux due to the way deferred memory allocation works. On Linux, the allocation can
+        //       succeed even if there is not enough memory but then the test may get killed by the OOM killer at the
+        //       time the memory is accessed which triggers the full memory allocation.
+        [Fact]
+        [Trait("MyTrait", "MyTraitValue")]
```

Comment:
Is this trait attribute necessary?

Reply:
changed to SortTrait, allows for easier testing of just Sort methods, just forgot to update this